src/main/scala/io/sdkman/changelogs/HelidonMigrations.scala

```diff
@@ -0,0 +1,23 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+import org.bson.Document
+
+@ChangeLog(order = "074")
+class HelidonMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001-add_helidon_candidate",
+    author = "who ?"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "helidon",
+      name = "Helidon",
+      description =
+        "Helidon is a cloud-native, open‑source set of Java libraries for writing microservices that run on a fast web core powered by Netty.",
```

Comment:
The Helidon CLI lets you easily create a Helidon project by picking from a set of archetypes. It also supports a developer loop that performs continuous compilation and application restart, so you can easily iterate over source code changes.